models/ExperimentFull.ts

```diff
@@ -218,3 +223,58 @@ export class ExperimentFull implements ApiDataSource {
     return !!this.endReason || !!this.conclusionUrl || typeof this.deployedVariationId === 'number'
   }
 }
+
+// Placing these here for now, but they should go in a schema.ts file
+//
+
+const idSchema = yup.number().integer().positive()
```

Comment:
```suggestion
const idSchema = yup.number().integer().min(0)
```
I am not sure where the IDs actually start in the database, but I know that the mock API is returning 0 for `ExperimentFull#deployedVariationId`.

Reply:
Good point, I have committed it :)

Reply:
It does seem like positive should be right as MariaDB auto-increments starting from 1.